android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -228,25 +216,28 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         editVisibleAssets.setOnClickListener(v -> {
             JsonRpcService jsonRpcService = getJsonRpcService();
             assert jsonRpcService != null;
-            jsonRpcService.getNetwork(CoinType.ETH, selectedNetwork -> {
-                EditVisibleAssetsBottomSheetDialogFragment bottomSheetDialogFragment =
-                        EditVisibleAssetsBottomSheetDialogFragment.newInstance(
-                                WalletCoinAdapter.AdapterType.EDIT_VISIBLE_ASSETS_LIST);
-
-                bottomSheetDialogFragment.setSelectedNetwork(selectedNetwork);
-                bottomSheetDialogFragment.setDismissListener(
-                        new EditVisibleAssetsBottomSheetDialogFragment.DismissListener() {
-                            @Override
-                            public void onDismiss(Boolean isAssetsListChanged) {
-                                if (isAssetsListChanged != null && isAssetsListChanged) {
-                                    updatePortfolioGetPendingTx(false);
-                                }
+            NetworkInfo selectedNetwork =
+                    mWalletModel.getCryptoModel().getNetworkModel().mDefaultNetwork.getValue();
+            if (selectedNetwork == null) {
+                return;
+            }
```

Comment:
nice!
nit: prefer to retain the [observed value, store it in reference](https://github.com/brave/brave-core/blob/062a8d984f6d38ca9c399994eea1f452edff6d0d/android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java#L169) and use it whenever it is needed.
Alternately, i have added [observeOnce](https://stackoverflow.com/a/54648758) in LiveDateUtils (new class) to get value only for single use.